examples/camera-app/linux/src/clusters/camera-avsettingsuserlevel-mgmt/camera-avsettingsuserlevel-manager.cpp

```diff
@@ -314,15 +314,26 @@ static void onTimerExpiry(System::Layer * systemLayer, void * data)
 {
     CameraAVSettingsUserLevelManager * delegate = reinterpret_cast<CameraAVSettingsUserLevelManager *>(data);
 
-    delegate->OnPhysicalMoveCompleted(Protocols::InteractionModel::Status::Success);
+    // Make sure that our delegate instance is still alive
+    if (delegate != nullptr)
```

Comment:
How does that make sure of anything?  The pointer is just the pointer.  You can't tell, by comparing it to null, whether the thing it points to is alive or dead.